spec/classes/jira_facts_spec.rb

```diff
@@ -24,7 +23,7 @@
           end
 
           it do
-            is_expected.to contain_file(pe_external_fact_file). \
+            is_expected.to contain_file(external_fact_file). \
```

Comment:
Perhaps this should be kept to a separate PR and remove the specifics for Puppet Enterprise? I think `$facts['aio_agent_version']` should also cover Puppet Enterprise. Then the specific testcase can also be removed.

Reply:
I see what you mean, that sounds good.